code/drasil-example/Drasil/GamePhysics/Unitals.hs

```diff
@@ -286,12 +287,17 @@ mmntOfInCons   = constrained' QP.momentOfInertia    [nonNegativeConstraint] (dbl
 gravAccelCons  = constrained' QP.gravitationalConst [] (dbl 9.8)
 posCons        = constrained' QP.position           [] (dbl 0.412) --FIXME: should be (0.412, 0.502) vector
 veloCons       = constrained' QP.velocity           [] (dbl 2.51)
-orientCons     = constrained' QM.orientation        [] (sy QM.pi_ / 2) -- physical constraint not needed space is radians
+orientCons     = constrained' QM.orientation        [physc $ Bounded (Inc, 0) (Inc, 2 * sy QM.pi_)] (sy QM.pi_ / 2) -- physical constraint not needed space is radians
 angVeloCons    = constrained' QP.angularVelocity    [] (dbl 2.1)
 forceCons      = constrained' QP.force              [] (dbl 98.1)
 torqueCons     = constrained' QP.torque             [] (dbl 200)
 restCoefCons   = constrained' QP.restitutionCoef    [physc $ Bounded (Inc,0) (Inc,1)] (dbl 0.8)
 
+posOutCons        = constrained' QP.position           [] (dbl 0.0)
```

Comment:
Also, it makes no sense to me to use `constrained'` with an empty list of constraints! That, in fact, should be a hard error.